packages/twenty-front/src/modules/app/components/SettingsRoutes.tsx

```diff
@@ -382,6 +398,14 @@ export const SettingsRoutes = ({
         path={SettingsPath.EditImapConnection}
         element={<SettingsEditImapConnection />}
       />
+      <Route
```

Comment:
Sorry you didn't have design but I didn't expect you to do two pages. For me since we decided that on the backend it was one account then the frontend should be the same, otherwise it will cause clear inconsistencies for users. 
We could have chosen to do 1 account for smtp, 1 account for caldav and 1 for imap on the backend - in that case your frontend would have been the right one. But it was not the path we took